keycloak/keycloak-admin-ui
# add priority attribute when dragging data list elements

## Motivation

Closes #603 

## Verification Steps


1. Go to Realm Settings -> Keys -> Providers list
2. Click and drag a provider to a new location
3. Inspect the network call and verify that the priority is changed based on the location in the providers list


## Checklist:
